modules/systemtests/src-test/org/ejbca/core/ejb/ca/caadmin/RenewCATest.java

```diff
@@ -97,6 +99,84 @@ public void test01renewCA() throws Exception {
         assertFalse(Arrays.equals(orgkey, newkey));
         log.trace("<test01renewCA()");
     }
+
+    
+    /** Test renewal of a CA using a different key algorithm. 
+     *  Note: Can run these tests alone by using: ant test:runone -Dtest.runone=RenewCATest
+    **/
+    
+    // Need these extra EJBs
+    private final CryptoTokenManagementSessionRemote cryptoTokenManagementSession = EjbRemoteHelper.INSTANCE
+            .getRemoteSession(CryptoTokenManagementSessionRemote.class);
+    private final CertificateProfileSessionRemote certificateProfileSession = EjbRemoteHelper.INSTANCE
+            .getRemoteSession(CertificateProfileSessionRemote.class);
+
+    @Test
+    public void test02renewCA_ChangeKeyAlg() throws Exception {
```

Comment:
No need to number it, i.e. 02, that is just old legacy from tests that needed to run sequentially. I think the ones in this file can run unordered, so just remove that.

Reply:
OK.